src/mintpy/utils/plot.py

```diff
@@ -1214,6 +1220,9 @@ def plot_gps(ax, SNWE, inps, metadata=dict(), print_msg=True):
             vprint(f'ignore the following {np.sum(nan_flag)} stations due to limited overlap/observations in time')
             vprint(f'  {site_names[nan_flag]}')
 
+        if 'UTM_ZONE' in metadata:
+            site_lats, site_lons = ut0.latlon2utm(site_lats, site_lons)
+
```

Comment:
My local testing based on the Ridgecrest example dataset as below did not display it right (all GPS are concentrated to the origin near 0), and moving this part of the code [here](https://github.com/insarlab/MintPy/blob/7ac345551ac237f0e1a314cd3f04593f4ea1305f/src/mintpy/utils/plot.py#L1136) fixed it.

```bash
view.py velocity.h5 velocity --show-gps
```